debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/TypeRegistry.java

```diff
@@ -136,24 +146,27 @@ public static Builder create(TypeInfo typeInfo) {
     private final int geometryOid;
     private final int geographyOid;
     private final int citextOid;
+    private final int inetOid;
```

Comment:
`inet` and `_inet` OIDs are constant so the should go into `PgOid` file - see https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_type.dat#L243

Reply:
Thanks! I'll fix it

Reply:
Fixed